### ISSUE TITLE
Phalcon\Session\Adapter\Stream   wrong open save path

### ISSUE THREAD

**Issue by ZhangRuiMingZRM** [phalcon/cphalcon#14265]
* Phalcon version: `4.0.0-beta.1`
* PHP Version: `7.3.6`
* Operating System: ubuntu
* Installation type: Compiling from source
* Server: php web server
   
When running the following code and get this:
```
 Warning: Phalcon\Session\Adapter\Stream::write(/90192p0avuirodu6r2914kvn9o): failed to open stream: Permission denied in Unknown on line 0
```
```php
    $session = new Manager();
    $files = new Stream(
        [
            'savePath' => '/tmp',
        ]
    );
    $session->setHandler($files);
    $session->start();
```
This may be due to the `public function open(var savePath, var sessionName)` in the `phalcon/phalcon/Session/Adapter/Stream.zep` 
https://github.com/phalcon/cphalcon/blob/4173bca9b85b4b7e3493146036ed91b977dff390/phalcon/Session/Adapter/Stream.zep#L97
```
    public function open(var savePath, var sessionName) -> bool
    {
        var path;

        if true !== ends_with(savePath, "/") {
            let path = savePath . "/";
        }

        let this->path = path;

        return true;
    }
```

Previous attributes `path` are  reassigned

**Comment by beareable** [comment]
what time this bug will be fixed?

**Comment by ruudboon** [comment]
Want to fix it before the next release. Bug is fixed in attached pull above. But I'm struggling with the tests. Need to fix that before merging.

**Comment by sergeyklay** [comment]
Fixed in the `4.0.x` branch. Feel free to open a new issue if the problem appears again. Thank you for the bug report.

**Comment by bnlambert** [comment]
Hi

I am currently having this same issue on 

CentOS Linux 7.8.2003
PHP 7.4.10 
Phalcon 7.4

It just suddenly happened no idea on how to fix it
